documentation/configuration.md

```diff
@@ -266,6 +265,21 @@ Match pid 1
 
 ## Metrics Configuration
 
+### Global Counter Interval
+
+Due to limitations in event counters, dotnet-monitor supports only **one** refresh interval when collecting metrics. This interval is used for
+Prometheus metrics, livemetrics, triggers, traces, and trigger actions that collect traces. The default interval is 5 seconds, but can be changed in configuration.
+
+```json
+{
+    "GlobalCounter": {
+      "IntervalSeconds": 10
+    }
+}
+```
+
+**Note:** As of Preview 8, Request Duration triggers do not match the default `Global Counter Interval`. Set the `Interval` to 10 seconds to enable this trigger.
```

Comment:
I suppose we don't have to go into specifics, such as that you don't need to match the 10 second interval (if it's lower, then you won't get runaway request detection; if its higher, then you won't get any triggering :smile:)